include/cantera/thermo/WaterSSTP.h

```diff
@@ -140,9 +140,11 @@ class WaterSSTP : public SingleSpeciesTP
     explicit WaterSSTP(XML_Node& phaseRef, const std::string& id = "");
 
     virtual std::string type() const {
-        return "Water";
+        return "WaterSSTP";
```

Comment:
I'm ok with going away from acronyms, although I'm not sure that this clearly differentiates from `PDSS_Water` (the `TP` portion did point at one feature of this class that makes it suited for the two-phase region).

Going forward, I do not know the prospects of `iawps-95` - if it's useful enough to keep around I hope to see it as a more accurate and fully featured implementation than `reynolds`, - in which case the `liquid` is somewhat limiting. Since the `TP` approach pre-empts extensions to two-phase it will require a different approach with a different class.